documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,4 +2,237 @@
+- A **valid milestone** must exist for the declaration.
+- The system checks for duplicated declarations.
+- From academic year 2025-26, if the participant is a **mentor**, they can only declare a `started` or `completed` status.
+- The provided **evidence** must meet the necessary requirements.
+
+These validations help ensure accurate and compliant record declarations while preventing errors in the process.
+
+## Working with participant bands
+
+Given the data:
+
+- **Bands:**
+  - **Band A**: 0 to 2000 participants
+  - **Band B**: 2000 to 4000 participants
+  - **Band C**: 4000 participants and up to a maximum of 115% of the contract's target volume
+- **Previous participants**: 2100
+- **Current participants**: 2000
+- **Goal**: Distribute these 2000 participants into the bands.
+
+### Breakdown
+
+- **Previous Distribution:**
+  - The **first 2000** participants recruited in a cohort were in **Band A**.
+  - The **next 100** participants (out of 2100) moved into **Band B**.
+
+- **Current Distribution (2000 participants):**
+  - Since all 2000 participants fit within **Band A’s range (0–2000)**,
+    **Band A now has 0 participants**.
+  - The remaining participants (from 2000 onwards) move into **Band B and Band C**:
+    - **Band B** starts at 2000. Since we need to place 2000 participants:
+      - 1900 fit into **Band B** (since the previous 100 were already in Band B, we adjust).
+      - The remaining 100 go into **Band C**.
+
+### **Final Result:**
+| Band   | Participants |
+|--------|-------------|
+| Band A | 0           |
+| Band B | 1900        |
+| Band C | 100         |
+
+This ensures the correct redistribution of participants across the defined bands.
+
+## Output Fee vs. Service Fee
+
+### Output fees
+
+Declarations are paid upon submission. Output fees make up 60% of the price per participant, which lead providers can claim for participant training and submission of evidence of engagement via the API.
+
+### Service fees
+
+Service fees make up 40% of the price per participant and are paid to lead providers monthly over the period the participant is being trained, based on lead provider recruitment targets.
+
+### Breakdown of fees for ECTs:
+
+Example: How a price per participant of £1,000 is distributed between **Service Fees** and **Output Fees**:
+
+### **Total Cost per ECT participant**: **£1,000**
+- This cost is split into **Service Fees (40%)** and **Output Fees (60%)**.
+
+### **1. Service Fee (£400, 40%)**
+- This is a fixed payment based on a provider’s recruitment target. If a provider recruits less than 75% of their target, the service fee will be reduced.
+- The fee is distributed over time:
+  - **90% is paid over 29 months** for standard-length inductions.
+  - **10% is paid over 40 months** for non-standard-length inductions.
+
+For the first 29 months of a contract, the provider receives:
+
+- **3.103% of the total service fee each month** (1/29th of 90%).
```

Comment:
discussed in person: Nathan has supplied these numbers however the logic is not reflected in the calculators. This is being confirmed with contract managers, if anything needs to change we'll do it in subsequent prs